tests/test_kokkos.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for Kokkos bindings.
+"""
+import pytest
+
+from pennylane_lightning_kokkos import InitArguments
+
+
+class TestKokkos:
+    """Tests that Kokkos bindings work."""
+
+    def test_InitArguments_init(self):
```

Comment:
You are performing 3 tests (assertions) here.
Let's say we had 30 similar assertions to verify. What would be the best approach?
Can you think of a way to parametrize it?

Reply:
Since you are using pytest, I would use `@pytest.mark.parametrize` and write something like
```
    @pytest.mark.parametrize("num_threads", [1, 2, 5])
    def test_InitArguments_init(self, num_threads):
        """Tests that InitArguments is properly initialized."""
        args = InitArguments(num_threads)
        assert args.num_threads == num_threads
```

Reply:
Test parametrized.